### PR TITLE
KOGITO-369: [DMN Designer] User can not add an DT column constraint

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/KOGITO-369

Unit tests to be covered by https://issues.jboss.org/browse/KOGITO-406

### CODE REVIEW

**Comment by manstis** [comment]
@jomarko This PR is good for review now.

Who else would you like to review _kogito_ related changes?

In order to test this you can use `mvn clean install -Pkogito` and use the resulting `kie-wb-common-webapp-kogito-testing` webapp (or `kie-wb-common-webapp-kogito-runtime` and rebuild the VSCode/GitHub plugins...)

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-marshaller/src/main/java/org/kie/workbench/common/dmn/webapp/kogito/marshaller/jre/javax/xml/namespace/QName.java [hunk 1]
```diff
@@ -97,6 +98,35 @@ public String toString() {
...
+            return false;
+        }
+
+        final QName qName = (QName) o;
+
+        if (!getNamespaceURI().equals(qName.getNamespaceURI())) {
+            return false;
+        }
+        return getLocalPart().equals(qName.getLocalPart());
+    }
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-marshaller/src/main/java/org/kie/workbench/common/dmn/webapp/kogito/marshaller/jre/javax/xml/namespace/QName.java]: We could implement as:
```java
@Override
    public boolean equals(final Object o) {
        if (this == o) {
            return true;
        }
        if (!(o instanceof QName)) {
            return false;
        }
        QName qName = (QName) o;
        return Objects.equals(getNamespaceURI(), qName.getNamespaceURI()) &&
                Objects.equals(getLocalPart(), qName.getLocalPart());
    }

We used this approach here https://github.com/kiegroup/drools/pull/2587
```

**Comment by jomarko** [comment]
### Manual Review
Failed, see the video where are two input clauses in an action.
https://issues.jboss.org/secure/attachment/12456808/two-inputs.webm

**Comment by manstis** [comment]
@jomarko Your manual review failed due to https://issues.jboss.org/browse/KOGITO-368

I tried with two columns both with "Expression" as the constraint type. Switching between two columns preserved each's constraint. I also saved and re-opened and constraints were correctly preserved. Can you re-test just with "Expression" type constraints please?

I'll update the `equals(..)` method as requested.

**Comment by jomarko** [comment]
@manstis I will retest, once you push changes with equals.

**Comment by manstis** [comment]
@jomarko New commit with update to `equals(..)` method implementation.

**Comment by manstis** [comment]
Jenkins please retest this.

**Pull request merged**